lib/core/presentation/pages/ai/widgets/ai_chat_command_section.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:app/gen/fonts.gen.dart';
+import 'package:app/theme/color.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class AiChatCommandSection extends StatelessWidget {
+  final String sectionLabel;
+  final List<Map<String, dynamic>> commandList;
```

Comment:
wonder why dynamic value ?  let create an entity for this
recommend => AiCommand